cf/domain.py

```diff
@@ -156,6 +158,79 @@ def close(self):
             removed_at="5.0.0",
         )  # pragma: no cover
 
+    @classmethod
+    def create(cls, x, y, dx, dy):
+        '''
+        Create a new domain with the given longitude and latitude bounds and intervals.
+        
+        :Parameters:
+        
+            x: tuple
+                A tuple of two numeric values representing the lower and upper bounds of the longitude dimension.
+        
+            y: tuple
+                A tuple of two numeric values representing the lower and upper bounds of the latitude dimension.
+        
+            dx: float
+                The interval between points along the longitude axis.
+        
+            dy: float
+                The interval between points along the latitude axis.
```

Comment:
Need some info on what happens if dy < 0

Reply:
True, the array will be empty. Should I add a check for negative `dy` and reverse the order of the array?
```python
if dy < 0:
    y_points = np.flip(y_points)